scripts/preprocessing/04b-run_ssp.py

```diff
@@ -52,14 +53,17 @@ def run_ssp(cfg, subject, session=None):
         raw_fname_in.update(split='01')
 
     raw = mne.io.read_raw_fif(raw_fname_in)
-    # XXX : n_xxx should be options in config
     msg = 'Computing SSPs for ECG'
     logger.debug(gen_log_message(message=msg, step=4, subject=subject,
                                  session=session))
 
+    reject_ecg_ = cfg.get_ssp_reject(
```

Comment:
```suggestion
    reject_ecg_ = config.get_ssp_reject(
```
Why the underscore?

Reply:
to me the underscore suggest is a temp / local variable just used below. It's just a visual hint